StackExchange.Redis/StackExchange/Redis/KeyspaceIsolation/DatabaseWrapper.cs

```diff
@@ -27,6 +27,33 @@ public RedisValue DebugObject(RedisKey key, CommandFlags flags = CommandFlags.No
             return Inner.DebugObject(ToInner(key), flags);
         }
 
+          public long GeoAdd(RedisKey key, RedisValue[] values, CommandFlags flags = CommandFlags.None)
+        {
+            return Inner.GeoAdd(ToInner(key), values, flags);
+        }
+
+        public RedisValue GeoDist(RedisKey key, RedisValue value0, RedisValue value1, GeoUnit geoUnit = GeoUnit.m,
+            CommandFlags flags = CommandFlags.None)
+        {
+            return Inner.GeoDist(ToInner(key), value0, value1, geoUnit, flags);
+        }
+
+
+        public bool GeoAdd(RedisKey key, RedisValue value0, RedisValue value1, RedisValue value2, CommandFlags flags = CommandFlags.None)
+        {
+            return Inner.GeoAdd(ToInner(key), value0,value1,value2, flags);
+        }
+
+        public RedisValue[] GeoHash(RedisKey key, RedisValue[] value, CommandFlags flags = CommandFlags.None)
+        {
+            return Inner.GeoHash(key, value, flags);
+        }
+
+        public RedisValue[,] GeoPos(RedisKey key, RedisValue[] value, CommandFlags flags = CommandFlags.None)
+        {
+            return Inner.GeoPos(key, value, flags);
+        }
+
         public double HashDecrement(RedisKey key, RedisValue hashField, double value, CommandFlags flags = CommandFlags.None)

```

Comment:
(note: also the async API surface, but: definitely worth getting the sync surface nailed before duplicating everything)